src/app/pages/credentials/new-users/all-users/user-details/user-access-card/user-access-card.component.html

```diff
@@ -0,0 +1,119 @@
+<mat-card class="card">
+  <mat-card-header>
+    <h3 mat-card-title>
+      {{ 'Access' | translate }}
+    </h3>
+  </mat-card-header>
+
+  @if (user(); as user) {
+    <mat-card-content>
+      <div class="content-wrapper last-login">
+        <span class="label">
+          {{ 'Last Login' | translate }}:
+          {{ 'N/A' | translate }}
```

Comment:
Why is `N/A' permanent here? Is the info not available from middleware yet?

Reply:
Yes, info is not available yet